power-sequencer/runtime_monitor.cpp

```diff
@@ -33,7 +33,7 @@ using namespace sdbusplus::org::open_power::Witherspoon::Fault::Error;
 
 int RuntimeMonitor::run()
 {
-#ifdef UCD90160_DEVICE_ACCESS
+#if defined UCD90160_DEVICE_ACCESS || defined MIHAWKCPLD_DEVICE_ACCESS
```

Comment:
That UCD90160 flag was never that useful, so you are welcome to remove  the MIHAWK equivalent and also do device access.

Reply:
@spinler 
ok, I got it.
I'll fix it.